lib/autocroak.xs

```diff
@@ -78,6 +81,11 @@ static OP* croak_##TYPE(pTHX) {\
 	return next;\
 }
 
+#define _ISNT_NEG1(val) (SvIV(val) != -1)
+#define _SSELECT_NFOUND ((GIMME_V == G_ARRAY) ? *(SP - 1) : TOPs)
```

Comment:
I think `ST(0)` is what you're looking for here

Reply:
Is that your preference? I can rewrite it however.

Reply:
Actually, using ST(0) would seem to necessitate a bit more duplication of logic since, as you point out, there’d need to be an `ax` set up.

Would you accept it as-is?

Reply:
I've just pushed a different implementation. If you could rebase the tests I'll merge all of those.

Reply:
Done.

What is `dAXMARKI`? I don’t see it in perl5.